Dockerfile

```diff
@@ -41,18 +42,17 @@ RUN conda config --add channels conda-forge && \
     bioblend galaxy-ie-helpers \
     # Jupyter widgets
     jupytext \
-    cython patsy statsmodels cloudpickle dill r-xml && conda clean -yt && \
+    source \ 
+    folium \
+    ftputil \ 
+    matplotlib \
+    altair \
+    shapely \
+    cython patsy statsmodels cloudpickle dill r-xml && \
+    conda clean -yt && \
     pip install jupyterlab_hdf
-    
-RUN conda install -c conda-forge source
```

Comment:
Please read about Dockerfiles when you have some time.

Having multiple RUN commands is less efficient. Every RUN command will have a "commit" (comparable to git commit). This means that once you made a commit all files are in the history of the image and can not be removed in any following commit. So its useful to aggregate commits and clean up in the same RUN command.

Reply:
Yes, I think it could be a good idea to read more about dockerfile indeed. Thank you for your help and infos !